bin/start_sync_stop.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+"""A script to test rippled in an infinite loop of start-sync-stop.
+
+- Expects 'rippled' to be on the `PATH` and 'rippled.cfg' to be in the current
+  directory.
+- Takes no arguments.
+- Can be stopped with SIGINT.
+- Tested on Python 3.8.
+- Has no dependencies outside the standard library.
+"""
+
+import asyncio
+import contextlib
+import json
+import logging
+import os
+import subprocess
+import time
+import urllib.error
+import urllib.request
+
+
+@contextlib.asynccontextmanager
+async def rippled(path='rippled', config='rippled.cfg'):
+    """A context manager for a rippled process."""
+    # Start the server.
+    process = await asyncio.create_subprocess_exec(
+        path,
+        '--conf',
+        config,
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL,
+    )
+    logging.info(f'rippled started with pid {process.pid}')
```

Comment:
_You_ know the `<num>` in `debug.<num>.log` indicates the number of iterations.  But that doesn't make it obvious to the rest of us.  Consider passing the iteration as an argument to `rippled()`, then print it here.  Perhaps something like...
```
logging.info(f'iteration {iteration} started. rippled pid: {process.pid}')
```